airflow/contrib/operators/azure_container_instances_operator.py

```diff
@@ -154,16 +204,24 @@ def execute(self, context):
                                                 share_name,
                                                 account_name,
                                                 read_only))
-            volume_mounts.append(VolumeMount(mount_name, mount_path, read_only))
+            volume_mounts.append(VolumeMount(name=mount_name,
+                                             mount_path=mount_path,
+                                             read_only=read_only))
 
         exit_code = 1
         try:
             self.log.info("Starting container group with %.1f cpu %.1f mem",
                           self.cpu, self.memory_in_gb)
 
+            gpu_resource = None
+            if self.gpu:
+                gpu_resource = GpuResource(count=self.gpu[0], sku=self.gpu[1])
```

Comment:
This is a named tuple, so we can do this right?
```suggestion
                gpu_resource = GpuResource(count=self.gpu.count, sku=self.gpu.sku)
```

Reply:
This requires the user to actually import and use the named tuple. Right now the user can choose to use the named tuple or just a tuple.

Should we force the user to import the named tuple? The current examples for the ACI class uses regular tuples.

Reply:
Ah, allow a normal tuple for now, but perhaps issue a deprecation warning (in the `__init__` if it is a plain tuple.) and upgrade the object in place?

Reply:
If we're going to move away from tuples (implying the user has to import classes), shouldn't we consider using Azure's model object as parameters instead? Right now we're pretty much just forwarding the parameters into the Azure model objects. If in the future Azure adds new functionality into their model object we wouldn't need to upgrade anything on our side.

Reply:
Yeah that sounds sensible.

Reply:
Using the azure GpuResource model as the parameter type. The operator doesn't currently support gpu, so no need to add a deprecation warning. Those who want to use it should import the model.